### DISCUSSION TITLE
2022/12/20/Strange_isn't_it

### DISCUSSION THREAD

**Discussion by giscus** [KMIT-Students/blog-demo#20] [General]
# 2022/12/20/Strange_isn't_it

What’s the strangest thing you have seen? My answer - Human Being. Why? Let’s say you are a kind person concerned about homeless people and you just won a lottery of $1,000,000 which leads to two scenarios. Scenario 1 - Everyone congratulating you for your luck. Scenario 2 - The only one ACTUALLY happy for you is yourself. Since you are a kind person you donated all the money you won to charity which leads to the birth of the worst case scenario that you never expected:Every finger around you pointing out to you as if you have committed an unforgivable crime. You soon will be referred to as the madman.You hear words like “Should’ve bought a house and a car” or “Why waste money?Clear your debts and start a new life.” Why?WHY must they be so judgemental aobut others’ decisions that they have nothing to do with? That is one of the many factors that makes me think the Human is the strangest “THING”.(Sorry if you expected Being) I’m thankful that I finally am free from such environment…

https://kmit-anubhav.github.io/2022/12/20/Strange_isn't_it.html

**Comment by MaruthiCharan2403** [comment]
cool and a good one
